app/components/find/hidden_fields_component.rb

```diff
@@ -0,0 +1,17 @@
+module Find
+  class HiddenFieldsComponent < ViewComponent::Base
+    attr_reader :query_params, :form_name, :form, :exclude_keys
+    # rubocop:disable Lint/MissingSuper
```

Comment:
Sure to super. Re the params, the previous PRs which created the forms, especially the locations form use different params to what is on find. I had a chat with @defong about this, and also the fact that on Find the back buttons generate extra params. It's complicated...either we stick with the new params or rewrite all the forms to duplicate what  is on find?

Reply:
ok let's have a quick huddle later to decide

Reply:
FYI spoke offline. We decided to stick with the original params to avoid any new cases we have to deal with. Will come back to this once ETP stuff is done.